bears/general/LineLengthBear.py

```diff
@@ -49,4 +67,6 @@ def run(self,
                     line=line_number + 1,
                     column=max_line_length + 1,
                     end_line=line_number + 1,
-                    end_column=len(line))
+                    end_column=len(line),
+                    aspect=Formatting('py'),
```

Comment:
Again, why `py` here?
Can it be omitted?

Need tests to validate what works vs what doesnt work, so we can see if any coalib changes are needed to make this implementation sensible.

Reply:
Currently, we have to pass the language here to create an object of the `Formatting` aspect. Otherwise, it pops an error, I will see if I can do some changes in the aspect core to omit this. I had explained the problem before https://gitter.im/coala/aspects?at=5b46f760b5efcd5aeef47912. But got no response.

Regarding the tests, `LocalBearTestHelper` needs to be modified for adding the tests. Let me know whether to do it here or in a separate PR?

Reply:
an approach you might like to try (test on a different fork; no PR necessary) is adding a bear setting for language : Language = Unknown , which you can pass to the Formatting aspect to make it happy.

`LocalBearTestHelper` is in a different repo, so it needs to be a separate PR.